entrypoint/fb.js

```diff
@@ -114,13 +113,11 @@ module.exports.message = (event, context, callback) => {
 module.exports.push = (event, context, callback = console.log) => {
   let params = null;
 
-  if (event.queryStringParameters !== undefined) {
-    params = event.queryStringParameters;  // Called from web
+  if ('timing' in event) {
+    params = event;
+  } else if (event.queryStringParameters && 'timing' in event.queryStringParameters) {
+    params = event.queryStringParameters;
```

Comment:
Best to just use a `timing` variable instead of dragging other parameters with you